libbeat/outputs/elasticsearch/client.go

```diff
@@ -82,12 +82,12 @@ type clientSettings struct {
 }
 
 type bulkResultStats struct {
-	acked        int // number of events ACKed by Elasticsearch
-	duplicates   int // number of events failed with `create` due to ID already being indexed
-	fails        int // number of events with retryable failures.
-	nonIndexable int // number of events with permanent failures.
-	deadLetter   int // number of failed events ingested to the dead letter index.
-	tooMany      int // number of events receiving HTTP 429 Too Many Requests
+	acked        []publisher.Event // number of events ACKed by Elasticsearch
```

Comment:
Have you quantified the potential memory impact of keeping the references to the events around longer?